src/__integration_tests__/postOrder.ts

```diff
@@ -35,15 +35,29 @@ const sdk = new OpenSeaSDK(
 );
 
 suite("SDK: order posting", () => {
+  test("Post Buy Order", async () => {
+    const postBuyOrder = {
+      accountAddress: WALLET_ADDRESS ? WALLET_ADDRESS : "",
+      startAmount: "0.004",
+      asset: {
+        tokenAddress: "0x1a92f7381b9f03921564a437210bb9396471050c",
+        tokenId: "2288",
+      },
+    };
+
+    await sdk.createBuyOrder(postBuyOrder);
```

Comment:
can we check that the order actually got posted? either in the response or do a 'get'

Reply:
currently just relying on it throwing an exception or not